setup.py

```diff
@@ -26,15 +26,45 @@
 
 
 extras_require = {
+    'client': [
+        'netifaces',
+        'posttroll>=1.5.1',
+        'pyinotify',
+        'pyzmq',
+        'trollsift',
+    ],
```

Comment:
Looks like ther requirements for client, mirror, server, dispatcher are pretty similar, do we really need to separate them?

Reply:
also, they seem to be the core of the package, why make them extra requirements? I can understand making optional for movers, since only the user know what's going to be used (s3, scp, paramiko)

Reply:
If there's another way for this I'm all ears. Client and dispatcher have the same requirements, as do mirror and server which require watchdog in addition to client and dispatcher.

Reply:
can't we have this simply as `install_requires`, including watchdog? Then we cover the base needs for 90% of the code...

Reply:
But we can't then use the S3 mover without installing all the unnecessary requirements. For cloud and microservices that's a no-go.

Reply:
aah, ok, so I suppose that's a sign that trollmoves and movers should be two separate packages...

Reply:
Yes, that might be one solution :thinking: